### ISSUE TITLE
Reorganize Databases section

### ISSUE THREAD

**Issue by SaschaAdler** [crkn-rcdr/Access-Platform#43]
In order to get this repo working with kivik, I'll either need to rework how kivik loads database content or reorganize the Databases directory. I think doing the latter makes some sense. The reorganization would work like this:

Current structure:
```
Databases
- schemas
- design-docs
- fixtures?
```

New structure:
```
Databases
- database 1
  - schema.json
  - design
  - fixtures
- database 2
...
- _util.schema.json
```

Alternatively we could expand the _util definitions into each individual schema file. While this wouldn't be the least repetitive approach it would solve issues pertaining to how the schema files can be used (#21).

Even if it's not strictly couch-related I'm planning on including some basic schema validation functionality in kivik as well.

**Comment by RussellMcOrmond** [comment]
As I don't know what problem this is solving, I can't really comment on the ticket or the related PR.  Proceed as you see fit, as long as Kivik will still work and be understandable.

**Comment by SaschaAdler** [comment]
Closed by #44.
